pkg/infra/pulumi_aws/deploylib.ts

```diff
@@ -62,7 +62,12 @@ export interface TopologyEdgeData {
     target: string
 }
 
-export const kloConfig = new pulumi.Config('klo')
+export interface RdsImport {
+    dbInstanceIdentifier: string
+    proxy: string
+}
+
+export const kloConfig: pulumi.Config = new pulumi.Config('klo')
```

Comment:
I'm surprised this needs the type hint

Reply:
doesnt need it, i was trying to get some other things working with config and must have just left it